chessboardeditor/src/main/java/no/bakkenbaeck/chessboardeditor/view/board/ChessBoardView.kt

```diff
@@ -164,4 +170,31 @@ class ChessBoardView @JvmOverloads constructor(
             cell.setPiece(pieceView)
         }
     }
+
+    override fun onMeasure(widthMeasureSpec: Int, heightMeasureSpec: Int) {
+        val (desiredWidth, desiredHeight) = getDesiredDimensions(widthMeasureSpec, heightMeasureSpec)
+        setMeasuredDimension(desiredWidth, desiredHeight)
+        setWidthHeight(desiredWidth, desiredHeight)
+        super.onMeasure(widthMeasureSpec, heightMeasureSpec)
+    }
+
+    private fun getDesiredDimensions(widthMeasureSpec: Int, heightMeasureSpec: Int): Pair<Int, Int> {
+        Log.v("ChessBoardView", "measured width ${MeasureSpec.toString(widthMeasureSpec)}")
+        Log.v("ChessBoardView", "measured height ${MeasureSpec.toString(heightMeasureSpec)}")
```

Comment:
Remove the `Logs`. Anyone using this library shouldn't have to see logs from this lib.

Reply:
👍